taxcalc/tests/test_policy.py

```diff
@@ -999,16 +972,24 @@ def test_raise_errors_regression():
     """
     This tests that raise_errors prevents the error from being thrown. The
     correct behavior is to exit the `adjust` function and store the errors.
+    ** From the parameters.py docstring for the adjust method:
+        raise_errors: Boolean
+          Raise errors as a ValidationError. If False, they will be stored
+          in the errors attribute.
     """
     ref = {
         "II_brk7-indexed": [{"value": True}],
         "II_brk6": [{"value": 316700, "MARS": "single", "year": 2020}],
         "II_brk7": [{"value": 445400, "MARS": "single", "year": 2020}],
 
     }
+    # MRH COMMENT ON 2025-08-04:
+    # I don't understand this test because the ref dictionary above seems OK
```

Comment:
The test is to see if a poorly specified reform raises errors and then that those errors are suppressed with `raise_errors=False`.

One the master branch`pol.errors` contains a dictionary that shows that `ref` is not consistent with non-adjusted current law parameters since at some point, `II_brk5` becomes higher than `II_brk6` (among other issues), which is not allowed.

What in this PR would change that result?

Reply:
@jdebacker,  I change the `test_raise_errors_regression` in `test_policy.py` so that the reform generates an error.